From: Marc Kleine-Budde <m.kleine-budde@pengutronix.de>
Subject: kexec-tools: fix static linking

When linking statically against libz, kexec-tools don't find symbols
like gzopen, because the linker evaluates libs from right to left. Using
LDLIBS instead of LDFLAGS fixes this issue.

Signed-off-by: Marc Kleine-Budde <m.kleine-budde@pengutronix.de>

---
 kexec/Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: kexec-tools-2.0.0/kexec/Makefile
===================================================================
--- kexec-tools-2.0.0.orig/kexec/Makefile
+++ kexec-tools-2.0.0/kexec/Makefile
@@ -67,10 +67,10 @@ KEXEC_MANPAGE = $(MANDIR)/man8/kexec.8
 
 -include $(KEXEC_DEPS)
 
-$(KEXEC): LDFLAGS+=$(LIBS)
+$(KEXEC): LDLIBS+=$(LIBS)
 $(KEXEC): $(KEXEC_OBJS) $(UTIL_LIB)
 	@$(MKDIR) -p $(@D)
-	$(LINK.o) -o $@ $^
+	$(LINK.o) -o $@ $^ $(LDLIBS)
 
 $(KEXEC): CPPFLAGS+=-I$(srcdir)/kexec/arch/$(ARCH)/include
 
